nimare/meta/cbma/ale.py

```diff
@@ -565,6 +661,31 @@ def __init__(
         # a Dataset with pre-generated MA maps is provided.
         self.memory_limit = "100mb"
 
+    def _generate_description(self):
```

Comment:
Here is an example SCALE description (no FWE correction). @angielaird WDYT?

>A specific coactivation likelihood estimation (SCALE) meta-analysis \citep{langner2014meta} was performed with NiMARE 0.0.12rc7+30.ga3c0f7b.dirty (RRID:SCR_017398; \citealt{Salo2022}), with 10 iterations. The input dataset included 267 foci from 21 experiments, with a total of 334.0 participants.

References

```bibtex
@article{Salo2022,
  doi = {10.55458/neurolibre.00007},
  url = {https://doi.org/10.55458/neurolibre.00007},
  year = {2022},
  publisher = {The Open Journal},
  volume = {1},
  number = {1},
  pages = {7},
  author = {Taylor Salo and Tal Yarkoni and Thomas E. Nichols and Jean-Baptiste Poline and Murat Bilgel and Katherine L. Bottenhorn and Dorota Jarecka and James D. Kent and Adam Kimbler and Dylan M. Nielson and Kendra M. Oudyk and Julio A. Peraza and Alexandre Pérez and Puck C. Reeders and Julio A. Yanes and Angela R. Laird},
  title = {NiMARE: Neuroimaging Meta-Analysis Research Environment},
  journal = {NeuroLibre}
}
@article{langner2014meta,
  title={Meta-analytic connectivity modeling revisited: controlling for activation base rates},
  author={Langner, Robert and Rottschy, Claudia and Laird, Angela R and Fox, Peter T and Eickhoff, Simon B},
  journal={Neuroimage},
  volume={99},
  pages={559--570},
  year={2014},
  publisher={Elsevier},
  url={https://doi.org/10.1016/j.neuroimage.2014.06.007},
  doi={10.1016/j.neuroimage.2014.06.007}
}
```